src/components/map/Map.tsx

```diff
@@ -50,61 +51,15 @@ const MapIcon = ({
   return <MarkerIcon color={iconColor} size={iconSize} />;
 };
 
-type TCluster = {
-  clusterColor: string;
-  geometry: { coordinates: [number, number] };
-  id: string;
-  onPress: () => void;
-  properties: { point_count: number };
-};
-
-const renderCluster = (cluster: TCluster) => {
-  const { clusterColor: backgroundColor, geometry, id, onPress, properties = {} } = cluster;
-  const { point_count: points } = properties;
-
-  return (
-    <Marker
-      key={`cluster-${id}`}
-      coordinate={{
-        longitude: geometry.coordinates[0],
-        latitude: geometry.coordinates[1]
-      }}
-      style={styles.clusterMarker}
-      onPress={onPress}
-    >
-      {CIRCLE_SIZES.map((size, index) => (
-        <View
-          key={`circle-${index}`}
-          style={[
-            styles.clusterCircle,
-            {
-              backgroundColor,
-              borderRadius: normalize(size / 2),
-              height: normalize(size),
-              opacity: 0.2 * (index + 1),
-              width: normalize(size)
-            }
-          ]}
-        >
-          <RegularText lightest center smallest>
-            {points}
-          </RegularText>
-        </View>
-      ))}
-    </Marker>
-  );
-};
-
-/* eslint-disable complexity */
 export const Map = ({
   calloutTextEnabled = false,
-  clusterDistance,
+  clusterDistance = 50,
   clusteringEnabled = false,
   geometryTourData,
   isMaximizeButtonVisible = false,
   isMultipleMarkersMap = false,
   isMyLocationButtonVisible = false,
-  locations,
+  locations = [],
```

Comment:
I see more than one unused prop here. Are these props here to be used later?

Reply:
you're right, not needed anymore https://github.com/smart-village-solutions/smart-village-app-app/pull/1154/commits/7d15a3d5f2af482f630518fdc922da1732420e58